tools/docker/docker-installation.md

```diff
@@ -8,11 +8,35 @@ order: 1
 
 ## Installation
 
-On Fedora 22 and above install the `docker` package:
+Install the `docker-ce` package using the Docker repository:
 
+To install the dnf-plugins-core package (which provides the commands to manage your DNF repositories) and set up the stable repository.
+
+```
+$ sudo dnf -y install dnf-plugins-core
+```
+To add the `docker-ce` repository
+
+```
+$ sudo dnf config-manager \
+    --add-repo \
+    https://download.docker.com/linux/fedora/docker-ce.repo
+```
+
+To install the docker engine 
+
+```
+$ sudo dnf install docker-ce docker-ce-cli containerd.io
 ```
-$ sudo dnf install docker
+
+Afterwards you need to enable the backward compatability for Cgroups
+
 ```
+sudo grubby --update-kernel=ALL --args="systemd.unified_cgroup_hierarchy=0"
+```
+
+> Note: You must reboot after running the command for the changes to take effect
```

Comment:
I'm not sure this works in jekyll. Could you remove it, as we'd like to have the 'guide' really straightforward, and this seems really important.

Reply:
Removed Quoted section (`>`) and Note. see 67dc784